### ISSUE TITLE
JOIN attempts not rate limited

### ISSUE THREAD

**Issue by badoge** [robotty/dank-twitch-irc#350]
Twitch recently fixed their rate limits to actually match the [documented limits](https://dev.twitch.tv/docs/irc/guide#rate-limits).

**Comment by RAnders00** [comment]
I will try to work on this within short-ish notice

**Comment by fmalk** [comment]
I was curious about this, so I looked at how [tmi.js](https://github.com/tmijs/tmi.js/blob/fae890153e8b3a2ab6bd56763f684606e0ebdb3c/lib/commands.js#L230) handles it.

It doesn't enforce rate limits at the lib level, if your bot exceeds it it will just receive an error.

**Comment by badoge** [comment]
Yeah I just made the bot join 1 channel per second to solve this problem since the bot is not verified and runs in 20+ channels.
